inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -372,13 +374,18 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
         VPU_LOGGER_SECTION(env.log);
 
         auto convertNetwork = [&convertedNetwork, &originalOrConvertNetwork]() {
+            // disable GeLU decomposition
+            const auto transformations_predicate = [](const std::shared_ptr<const ::ngraph::Node> &node) -> bool {
+                return std::dynamic_pointer_cast<const ::ngraph::opset2::Gelu>(node) != nullptr;
```

Comment:
Why are you using opset2 if opset3 is currently relevant?

Reply:
copy-paste)